cpp/include/cuspatial/experimental/hausdorff.cuh

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) 2022, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <cudf/types.hpp>
+#include <memory>
+
+#include <rmm/cuda_stream_view.hpp>
+
+namespace cuspatial {
+
+/**
+ * @brief Computes Hausdorff distances for all pairs in a collection of spaces
```

Comment:
We want to include this in the documentation too.

```suggestion
 * @ingroup distance
 * @brief Computes Hausdorff distances for all pairs in a collection of spaces
```

Also in `doxygen_groups.h`, we might want to add this file under `distance` group.

Reply:
Done.